packages/xo-server-usage-report/src/index.js

```diff
@@ -149,8 +149,8 @@ Handlebars.registerHelper(
     new Handlebars.SafeString(
       isFinite((value = round(value, 2))) && value !== 0
         ? value > 0
-          ? `(<b style="color: green;">▲ ${value}%</b>)`
-          : `(<b style="color: red;">▼ ${String(value).slice(1)}%</b>)`
+          ? `(<b style="color: green;">&#x25B2; ${value}%</b>)`
+          : `(<b style="color: red;">&#x25BC; ${String(value).slice(1)}%</b>)`
```

Comment:
Why this change which appears unrelated to the PR?

Reply:
Yes, it's unrelated but it was broken so I thought it would be harmless to fix that in the same time.

Reply:
Either do a separate PR or a separate commit in this PR and tell me to not squash the commits :slightly_smiling_face:

Reply:
I reverted the change